numpydoc/docscrape.py

```diff
@@ -715,6 +716,19 @@ def properties(self):
             )
         ]
 
+    @staticmethod
+    def _should_skip_member(name, klass):
+        if (
+            # Namedtuples should skip everything in their ._fields as the
+            # docstrings for each of the members is: "Alias for field number X"
```

Comment:
Does it work properly with https://github.com/numpy/numpydoc/issues/257#issuecomment-1002632904 ?

Reply:
Tested with code from [this comment](https://github.com/numpy/numpydoc/issues/257#issuecomment-1002632904), using

```
.. autoclass:: mypkg.Foo
    :members:
```

that outputs:

![image](https://github.com/numpy/numpydoc/assets/17479683/3c1b317b-271a-4a5d-88cb-40fb7e9d291a)


does that answer to the question?

Reply:
It does, thank you!